doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -479,7 +479,15 @@ Multi-image build
 To build and program a multi-image HEX file, follow the instructions in :ref:`programming_cmd` for the application core sample.
 
 To program the multi-image HEX file, you must use west.
-Programming multi-image builds for different cores is not yet supported in nrfjprog.
+You can program multi-image builds for different cores using the following command:
```

Comment:
I think we could we have commands for west also mentioned like we have for the Separate images above (in tabs)?

For west and for nrfjprog.

Reply:
Good idea, I will look into adding this

Reply:
West automatically programs all images in multi-image builds when using "west flash". That is: both app core and net core, as well as any bootloaders, TF-M and etc